cloudify_rest_client/manager.py

```diff
@@ -212,12 +202,30 @@ def add_manager(self, hostname, private_ip, public_ip, version,
         })
         return ManagerItem(response)
 
-    def get_managers(self, _include=None):
+    def remove_manager(self, hostname):
+        """
+        Remove a manager from the managers table
+
+        Will be used for clustering when a manager needs to be removed from
+        the cluster, not necessarily for uninstalling the manager
+        :param hostname: The manager's hostname
         """
-        Get all the managers in the managers table
+        response = self.api.delete('/managers', data={
+            'hostname': hostname
+        })
+        return ManagerItem(response)
+
+    def get_managers(self, hostname=None, _include=None):
```

Comment:
What can _include will contain here other them hostnames?
Can you use _include to pass the specific hostname to the query only?

Reply:
Include only specific columns in the returned value